mmdet3d/core/evaluation/kitti_utils/eval.py

```diff
@@ -715,21 +732,31 @@ def kitti_eval(gt_annos,
         curcls_name = class_to_name[curcls]
         for i in range(min_overlaps.shape[0]):
             # prepare results for print
-            result += ('{} AP@{:.2f}, {:.2f}, {:.2f}:\n'.format(
+            result += ('{} AP/AP40@{:.2f}, {:.2f}, {:.2f}:\n'.format(
```

Comment:
It will be clearer to print the AP and AP40 separately? I mean print AP bbox, bev, 3d, aos and then AP40 bbox, bev, 3d, aos?